torchaudio/prototype/models/hdemucs.py

```diff
@@ -469,13 +469,13 @@ def _spec(self, x):
         # which is not supported by torch.stft.
         # Having all convolution operations follow this convention allow to easily
         # align the time and frequency branches later on.
-        assert hl == nfft // 4
+        assert hl == nfft // 4, "hop length is nfft // 4"
         le = int(math.ceil(x.shape[-1] / hl))
         pad = hl // 2 * 3
         x = F.pad(x, [pad, pad + le * hl - x.shape[-1]], mode="reflect")
 
         z = _spectro(x, nfft, hl)[..., :-1, :]
-        assert z.shape[-1] == le + 4, (z.shape, x.shape, le)
+        assert z.shape[-1] == le + 4, "z.shape's last index must be 4 + le"
```

Comment:
can this be more descriptive, what are `z` and `le`? ex/ spectrogram's last dimension must be ...